tests/phpunit/tests/rest-api/rest-global-styles-controller.php

```diff
@@ -119,6 +119,11 @@ public function test_context_param() {
 		// Controller does not use get_context_param().
 	}
 
+	/**
+	 * @covers WP_REST_Global_Styles_Controller::get_theme_items
+	 * @ticket
+	 *
+	 */
 	public function test_get_theme_items() {
```

Comment:
These tests will fail and are WIP until https://github.com/WordPress/wordpress-develop/pull/6482 gets in.

https://github.com/WordPress/wordpress-develop/pull/6482 enables the feature in the first place 👍🏻